src/aggregate/functions/date.c

```diff
@@ -272,19 +272,19 @@ static int parseTime(ExprEval *ctx, RSValue *result, RSValue **argv, size_t argc
 
   char fmtbuf[1024] = {0};
   char valbuf[1024] = {0};
-  size_t fmtlen;
 
-  const char *origfmt = RSValue_StringPtrLen(argv[0], &fmtlen);
+  size_t fmtlen;
```

Comment:
better init this to 0

Reply:
`fmtlen` is used as an out param for `RSValue_StringPtrLen`.
If it fails, we exit the function so there is no meaning to initializing it.